### ISSUE TITLE
Determine people process for escalation procedures

### ISSUE THREAD

**Issue by clantosswett** [department-of-veterans-affairs/va.gov-team#827]
POC: 
Kimberly

User Story: 
As a call center lead, I need to have a process for issues to be escalated to my team that is controlled for quality and volume. 
 
Definition of Done:
-Determine people process for escalation at WHH
-Confirm with WHH team
-Roll out process to agents and supervisors 
-Conduct feedback and retro to determine any changes based on experience

**Comment by clantosswett** [comment]
After conversation with Lana about future salesforce development for VA.gov issues, Chante and Kimberley are going to write up the takeaways from the conversation and try to summarize our recommendations for moving forward. This will include the topic of technical/people processes for escalating VA.gov issues.

**Comment by clantosswett** [comment]
@Kimberley2019 thinking it might make sense to rework this issue. It's kind of big and will probably take a while (i.e. multiple sprints). Wondering if we should change this to an epic and create issues that are tied to the larger epic. Maybe we can discuss tomorrow during our meeting.

**Comment by Kimberley2019** [comment]
@clantosswett you are correct in your thoughts.  This is a big undertaking and should be broken down to a smaller epic so that achievable tasks can be created for that Sprint.  Yes, we should definitely discuss during our call tomorrow.

**Comment by Kimberley2019** [comment]
Kimberley will create a rough draft of the:  
Determine people process for escalation at WHH
Determine technical process for escalation at WHH
for Chante to review and update.
Due to Chante NLT 8/13/2019 COB.

**Comment by clantosswett** [comment]
Kimberley created a draft of the WHH-->Triage team escalation process. Has been shared with the WHH operations team and will be discussed and reviewed.

**Comment by Kimberley2019** [comment]
The process has been created by Kimberley, reviewed by @clantosswett, and sent to WHH leadership.  Meeting to review documents was scheduled for 8/13/2019 with WHH to review processes.  Meeting was postponed and new date is TBD.  Chante stated that she will reach out today (8/20/2019) for future date.

Cannot move forward until last three action items are completed.

Also, @clantosswett is this ticket the same as 857?

**Comment by Kimberley2019** [comment]
@clantosswett because of the progress that has been made, can this be moved to 'in progress' versus 'backlog'.
